[2023-08-15T18:19:58.318-0300] {processor.py:157} INFO - Started process (PID=18396) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:19:58.320-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:19:58.321-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:19:58.321-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:19:58.327-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:19:58.325-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:19:58.327-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:19:58.327-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:19:58.328-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:19:58.345-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-08-15T18:21:13.910-0300] {processor.py:157} INFO - Started process (PID=18868) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:21:13.912-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:21:13.913-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:21:13.913-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:21:13.920-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:21:13.919-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:21:13.921-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:21:13.921-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:21:13.921-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:21:13.940-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-08-15T18:22:10.305-0300] {processor.py:157} INFO - Started process (PID=19072) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:22:10.308-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:22:10.316-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:22:10.315-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:22:10.324-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:22:10.322-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:22:10.324-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:22:10.324-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:22:10.325-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:22:10.343-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-08-15T18:23:02.951-0300] {processor.py:157} INFO - Started process (PID=19259) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:23:02.953-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:23:02.955-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:23:02.954-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:23:02.962-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:23:02.960-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:23:02.962-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:23:02.962-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:23:02.963-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:23:02.981-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-08-15T18:24:03.224-0300] {processor.py:157} INFO - Started process (PID=19490) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:24:03.226-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:24:03.230-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:24:03.229-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:24:03.239-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:24:03.237-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:24:03.239-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:24:03.239-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:24:03.240-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:24:03.263-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2023-08-15T18:26:01.962-0300] {processor.py:157} INFO - Started process (PID=20538) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:26:01.976-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:26:01.977-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:26:01.977-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:26:02.001-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:26:01.996-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:26:02.001-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:26:02.001-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:26:02.002-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:26:02.032-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.082 seconds
[2023-08-15T18:27:22.673-0300] {processor.py:157} INFO - Started process (PID=20929) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:27:22.709-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:27:22.711-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:27:22.710-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:27:22.721-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:27:22.720-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:27:22.722-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:27:22.721-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:27:22.722-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:27:22.744-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.081 seconds
[2023-08-15T18:28:21.209-0300] {processor.py:157} INFO - Started process (PID=21158) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:28:21.221-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:28:21.222-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:28:21.222-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:28:21.228-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:28:21.227-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:28:21.229-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:28:21.228-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:28:21.229-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:28:21.249-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2023-08-15T18:29:19.859-0300] {processor.py:157} INFO - Started process (PID=21381) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:29:19.881-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:29:19.883-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:29:19.882-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:29:19.890-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:29:19.888-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:29:19.891-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:29:19.890-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:29:19.891-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:29:19.912-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2023-08-15T18:30:17.124-0300] {processor.py:157} INFO - Started process (PID=21600) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:30:17.139-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:30:17.141-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:30:17.140-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:30:17.147-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:30:17.146-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:30:17.148-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:30:17.148-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:30:17.148-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:30:17.167-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2023-08-15T18:31:13.195-0300] {processor.py:157} INFO - Started process (PID=21801) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:31:13.197-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:31:13.199-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:31:13.198-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:31:13.206-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:31:13.204-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:31:13.207-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:31:13.207-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:31:13.208-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:31:13.228-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2023-08-15T18:32:07.810-0300] {processor.py:157} INFO - Started process (PID=22001) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:32:07.818-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:32:07.820-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:32:07.819-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:32:07.827-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:32:07.825-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:32:07.827-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:32:07.827-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:32:07.828-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:32:07.848-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-08-15T18:33:04.253-0300] {processor.py:157} INFO - Started process (PID=22217) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:33:04.263-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:33:04.264-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:33:04.264-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:33:04.271-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:33:04.269-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:33:04.271-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:33:04.271-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:33:04.272-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:33:04.289-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2023-08-15T18:34:02.730-0300] {processor.py:157} INFO - Started process (PID=22434) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:34:02.750-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:34:02.751-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:34:02.751-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:34:02.758-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:34:02.756-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:34:02.758-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:34:02.758-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:34:02.759-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:34:02.777-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2023-08-15T18:34:56.630-0300] {processor.py:157} INFO - Started process (PID=22624) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:34:56.649-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:34:56.651-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:34:56.650-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:34:56.664-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:34:56.655-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:34:56.665-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:34:56.665-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:34:56.665-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:34:56.688-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2023-08-15T18:35:51.983-0300] {processor.py:157} INFO - Started process (PID=22822) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:35:51.985-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:35:51.987-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:35:51.986-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:35:51.992-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:35:51.991-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:35:51.993-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:35:51.993-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:35:51.993-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:35:52.013-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-08-15T18:36:50.992-0300] {processor.py:157} INFO - Started process (PID=23041) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:36:50.994-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:36:50.995-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:36:50.995-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:36:51.001-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:36:51.000-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:36:51.001-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:36:51.001-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:36:51.002-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:36:51.018-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-08-15T18:38:12.670-0300] {processor.py:157} INFO - Started process (PID=23371) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:38:12.674-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:38:12.676-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:38:12.675-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:38:12.687-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:38:12.685-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:38:12.687-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:38:12.687-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:38:12.692-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:38:12.725-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2023-08-15T18:44:37.833-0300] {processor.py:157} INFO - Started process (PID=23821) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:44:37.852-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:44:37.854-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:44:37.853-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:44:37.859-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:44:37.858-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:44:37.860-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:44:37.859-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:44:37.860-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:44:37.878-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2023-08-15T18:46:02.404-0300] {processor.py:157} INFO - Started process (PID=24208) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:46:02.413-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:46:02.415-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:46:02.414-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:46:02.420-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:46:02.419-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:46:02.421-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:46:02.421-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:46:02.421-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:46:02.439-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2023-08-15T18:47:06.787-0300] {processor.py:157} INFO - Started process (PID=24432) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:47:06.790-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:47:06.792-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:47:06.791-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:47:06.798-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:47:06.797-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:47:06.798-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:47:06.798-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:47:06.799-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:47:06.827-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2023-08-15T18:48:10.325-0300] {processor.py:157} INFO - Started process (PID=24657) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:48:10.329-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:48:10.331-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:48:10.330-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:48:10.338-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:48:10.336-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:48:10.350-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:48:10.350-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:48:10.350-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:48:10.395-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2023-08-15T18:49:18.725-0300] {processor.py:157} INFO - Started process (PID=24880) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:49:18.737-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:49:18.739-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:49:18.738-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:49:18.746-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:49:18.743-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:49:18.746-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:49:18.746-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:49:18.747-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:49:18.770-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2023-08-15T18:50:17.481-0300] {processor.py:157} INFO - Started process (PID=25093) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:50:17.482-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:50:17.485-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:50:17.484-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:50:17.491-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:50:17.490-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:50:17.492-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:50:17.491-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:50:17.492-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:50:17.518-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2023-08-15T18:51:26.397-0300] {processor.py:157} INFO - Started process (PID=25382) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:51:26.399-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:51:26.400-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:51:26.400-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:51:26.406-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:51:26.405-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:51:26.406-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:51:26.406-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:51:26.407-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:51:26.424-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-08-15T18:52:32.841-0300] {processor.py:157} INFO - Started process (PID=25753) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:52:32.854-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:52:32.856-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:52:32.855-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:52:32.863-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:52:32.861-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:52:32.863-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:52:32.863-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:52:32.864-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:52:32.882-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2023-08-15T18:53:30.050-0300] {processor.py:157} INFO - Started process (PID=25959) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:53:30.061-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:53:30.062-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:53:30.062-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:53:30.069-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:53:30.067-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:53:30.070-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:53:30.070-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:53:30.071-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:53:30.094-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2023-08-15T18:54:37.152-0300] {processor.py:157} INFO - Started process (PID=26234) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:54:37.177-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:54:37.179-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:54:37.178-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:54:37.186-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:54:37.184-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:54:37.186-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:54:37.186-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:54:37.187-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:54:37.206-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2023-08-15T18:56:52.727-0300] {processor.py:157} INFO - Started process (PID=26904) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:56:52.937-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T18:56:52.940-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:56:52.938-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:56:53.046-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:56:53.044-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T18:56:53.047-0300] {logging_mixin.py:150} INFO - [2023-08-15T18:56:53.047-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T18:56:53.047-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T18:56:53.071-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.349 seconds
[2023-08-15T19:01:53.948-0300] {processor.py:157} INFO - Started process (PID=27628) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:01:53.961-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:01:53.963-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:01:53.962-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:01:54.006-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:01:53.994-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:01:54.007-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:01:54.007-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:01:54.008-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:01:54.035-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.093 seconds
[2023-08-15T19:03:35.200-0300] {processor.py:157} INFO - Started process (PID=28059) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:03:35.213-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:03:35.214-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:03:35.213-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:03:35.224-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:03:35.219-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:03:35.225-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:03:35.224-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:03:35.225-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:03:35.266-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.079 seconds
[2023-08-15T19:05:09.893-0300] {processor.py:157} INFO - Started process (PID=28533) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:05:09.895-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:05:09.897-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:05:09.896-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:05:09.903-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:05:09.902-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:05:09.904-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:05:09.904-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:05:09.905-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:05:09.923-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-08-15T19:06:21.078-0300] {processor.py:157} INFO - Started process (PID=28765) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:06:21.080-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:06:21.082-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:06:21.081-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:06:21.089-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:06:21.088-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:06:21.090-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:06:21.090-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:06:21.091-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:06:21.113-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2023-08-15T19:07:21.980-0300] {processor.py:157} INFO - Started process (PID=28929) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:07:21.997-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:07:21.999-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:07:21.999-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:07:22.006-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:07:22.005-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:07:22.007-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:07:22.007-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:07:22.007-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:07:22.027-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2023-08-15T19:08:31.119-0300] {processor.py:157} INFO - Started process (PID=29207) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:08:31.128-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:08:31.129-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:08:31.129-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:08:31.135-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:08:31.134-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:08:31.136-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:08:31.135-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:08:31.136-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:08:31.157-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-08-15T19:09:31.872-0300] {processor.py:157} INFO - Started process (PID=29365) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:09:31.891-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:09:31.892-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:09:31.891-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:09:31.898-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:09:31.897-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:09:31.899-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:09:31.899-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:09:31.899-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:09:31.924-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.061 seconds
[2023-08-15T19:10:31.582-0300] {processor.py:157} INFO - Started process (PID=29523) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:10:31.584-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:10:31.585-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:10:31.585-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:10:31.594-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:10:31.590-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:10:31.594-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:10:31.594-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:10:31.595-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:10:31.637-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.061 seconds
[2023-08-15T19:11:32.361-0300] {processor.py:157} INFO - Started process (PID=29752) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:11:32.381-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:11:32.383-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:11:32.382-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:11:32.415-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:11:32.389-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:11:32.416-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:11:32.416-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:11:32.417-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:11:32.438-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.083 seconds
[2023-08-15T19:12:38.292-0300] {processor.py:157} INFO - Started process (PID=29953) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:12:38.298-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:12:38.300-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:12:38.299-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:12:38.306-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:12:38.305-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:12:38.307-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:12:38.307-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:12:38.307-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:12:38.325-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2023-08-15T19:13:38.069-0300] {processor.py:157} INFO - Started process (PID=30114) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:13:38.087-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:13:38.089-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:13:38.088-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:13:38.100-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:13:38.096-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:13:38.100-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:13:38.100-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:13:38.101-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:13:38.147-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.094 seconds
[2023-08-15T19:14:38.585-0300] {processor.py:157} INFO - Started process (PID=30267) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:14:38.587-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:14:38.589-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:14:38.588-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:14:38.595-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:14:38.593-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:14:38.595-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:14:38.595-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:14:38.596-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:14:38.613-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-08-15T19:15:34.352-0300] {processor.py:157} INFO - Started process (PID=30413) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:15:34.363-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:15:34.364-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:15:34.364-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:15:34.371-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:15:34.369-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:15:34.371-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:15:34.371-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:15:34.375-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:15:34.398-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2023-08-15T19:16:33.117-0300] {processor.py:157} INFO - Started process (PID=30569) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:16:33.123-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:16:33.125-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:16:33.124-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:16:33.131-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:16:33.129-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:16:33.131-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:16:33.131-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:16:33.132-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:16:33.154-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-08-15T19:17:34.616-0300] {processor.py:157} INFO - Started process (PID=30731) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:17:34.629-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:17:34.631-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:17:34.631-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:17:34.639-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:17:34.638-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:17:34.640-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:17:34.640-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:17:34.641-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:17:34.661-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2023-08-15T19:18:34.141-0300] {processor.py:157} INFO - Started process (PID=30883) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:18:34.153-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:18:34.155-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:18:34.154-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:18:34.162-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:18:34.160-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:18:34.162-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:18:34.162-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:18:34.163-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:18:34.183-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2023-08-15T19:19:29.686-0300] {processor.py:157} INFO - Started process (PID=31024) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:19:29.699-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:19:29.701-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:19:29.700-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:19:29.709-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:19:29.707-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:19:29.709-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:19:29.709-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:19:29.710-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:19:29.728-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2023-08-15T19:20:32.960-0300] {processor.py:157} INFO - Started process (PID=31249) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:20:32.968-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:20:32.970-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:20:32.969-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:20:32.977-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:20:32.975-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:20:32.977-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:20:32.977-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:20:32.978-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:20:32.996-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2023-08-15T19:21:46.511-0300] {processor.py:157} INFO - Started process (PID=31587) to work on /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:21:46.528-0300] {processor.py:826} INFO - Processing file /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-15T19:21:46.529-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:21:46.529-0300] {dagbag.py:541} INFO - Filling up the DagBag from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:21:46.537-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:21:46.535-0300] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-15T19:21:46.537-0300] {logging_mixin.py:150} INFO - [2023-08-15T19:21:46.537-0300] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-15T19:21:46.538-0300] {processor.py:838} WARNING - No viable dags retrieved from /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-15T19:21:46.556-0300] {processor.py:179} INFO - Processing /home/alan/repos/pipeline_airflow/pipeline-airflow-venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
